CHANGELOG.md

```diff
@@ -0,0 +1,525 @@
+# Changelog
+
+## v5.0.0
+Released by Jordan Harband on 2020-04-24T21:40:59Z
+
+v5.0.0
```

Comment:
can we avoid mentioning the version number twice?

Reply:
This came from the tag's corresponding commit message. The heading comes from the tag name and the body is the commit message referenced by the tag. I'll see what I can do.